packages/contracts/tasks/helpers/ContractStorage.ts

```diff
@@ -206,13 +206,12 @@ export class ContractStorage {
     network: string,
     keys: string[] = [],
   ): (string | undefined)[] {
-    const collection = this.db.get(`${network}.named`).value() as
-      | Record<string, IStorageNamedEntry | Record<string, IStorageNamedEntry>>
-      | undefined;
-
     return ids
       .map((id, index) => {
-        const entry = collection?.[id];
+        const entry = this.db.get(`${network}.named.${id}`).value() as
```

Comment:
can you explain what this change does?

Reply:
This is a main fix for factory reusing. `id` can contain dots (`.`) and lowdb uses dots to create new property inside.
For example, `a.b` id will be transformed to `{ "a": { "b": ... } }` object.
In case, if I use `get` api method for lowdb, it doesn't matter what is the internal structure for the storage and it gets the correct data (in our case policy and checker factories).

Reply:
right ok, that was helpful cheers